src/pages/Profile/index.tsx

```diff
@@ -219,10 +221,13 @@ const mapStateToProps: MapStateToProps<StateProps, OwnProps, AppState> = (
   ownProps
 ) => {
   const loggedInMember = getLoggedInMember(state);
-  const profileMember = state.membersNew.byMid[ownProps.match.params.memberMid];
+  const memberUsername = ownProps.match.params.memberMid;
```

Comment:
Why is `memberUsername` in the Redux `mapStateToProps` if it's just directly pulled from `ownProps`? It can just be a local variable in the render method, no need to duplicate it in multiple places.